optimum/exporters/openvino/__main__.py

```diff
@@ -339,7 +339,10 @@ def main_export(
                 orig_post_init_model = GPTQQuantizer.post_init_model
 
                 def post_init_model(self, model):
-                    from auto_gptq import exllama_set_max_input_length
+                    try:
+                        from gptqmodel import exllama_set_max_input_length
+                    except ImportError:
+                        from auto_gptq import exllama_set_max_input_length
```

Comment:
let's use an if else statement here, using is_auto_gptq_available , is_gptqmodel_available, else raise an errror "neither auto_gptq nor gptqmodel is avaialable".